docker_explorer/de.py

```diff
@@ -231,45 +232,44 @@ def Mount(self, container_id, mountpoint):
     container_object = self.GetContainer(container_id)
     container_object.Mount(mountpoint)
 
-  def GetContainersString(self, only_running=False):
-    """Returns a string describing the running containers.
+  def GetContainersJson(self, only_running=False):
+    """Returns a JSON string describing the running containers.
```

Comment:
The function seems to return an object (e.g. list) rather than a string. This is also what the `Returns:` statement says

Reply:
done